### DISCUSSION TITLE
Is it possible to custom columns ?

### DISCUSSION THREAD

**Discussion by a-peltier-edv** [dani-garcia/vaultwarden#5061] [General]
Hello, I started to deploy Vaultwarden on my debian server.
It works great and I have one question.

Is it possible to custom displayed columns ?
![image](https://github.com/user-attachments/assets/4b3e8534-2bc0-4b2e-86dd-30c553114855)

Actually we only see "Name, Groups, Permissions" and some users want to add an URL column or user column.

Thanks in advance.

**Comment by BlackDex** [comment]
No, this is not possible.
